apps.awesim.org/apps/myjobs/templates/Basic_BLAST_Job_Owens/blast.sh

```diff
@@ -1,5 +1,5 @@
 #PBS -l nodes=1:ppn=1
-#PBS -l walltime=10:00
+#PBS -l walltime=1:10:00
```

Comment:
Can this run under an hour? If so it should be set to something like `00:45:00` so that debug queue would be an option.